src/app/api/staking/getStakedDeposits.ts

```diff
@@ -0,0 +1,43 @@
+import Tenant from "@/lib/tenant/tenant";
```

Comment:
for naming consistency sake the other route is `getDeposit` but this is `getStakedDeposits` maybe we should make the other one `getStakedDeposit` but this is a nit and not necessary